### PR TITLE
Python update readme and setup requirements

### PR DESCRIPTION
**Pull request by joeldodge79**
Found a bug for folks who only use pip: they were getting cattrs=0.9.0
instead of the cattrs=1.0.0rc0. Adding the minimum requirement in
setup.py fixes this.

Also tweaked the "sample project setup" documentation to be more
accurate for installing python 3.7 via pyenv as well as making sure the
pipenv environment is created using the correct version of python.

### CODE REVIEW

File: python/README.rst [hunk 1]
```diff
@@ -36,14 +53,13 @@ Install python3.7 and use it for this directory
...
     cd looker-sdk-example/
-    pyenv install 3.7.4
     pyenv local 3.7.4
 
 Install looker_sdk using pipenv
 
 .. code-block:: bash
 
-    $ pipenv install --pre looker_sdk
+    pipenv install --python 3.7 --pre looker_sdk
```
Comment (josephaxisa) [inline python/README.rst]: What do you think of splitting this into two commands? Clearer, unnecessary or more confusing? 
```
pipenv shell --python 3.7.4
pipenv install --pre looker_sdk
```

File: python/README.rst [hunk 2]
```diff
@@ -16,14 +16,31 @@ and please indicate which language SDK you're using in the report.
...
 .. code-block:: bash
 
-    brew update && brew install pyenv && brew install pipenv
+    brew install pyenv
+
+Follow the **remaining steps 3 - 5** of
+https://github.com/pyenv/pyenv#basic-github-checkout otherwise your python3.7
+installation may break.
+
+Now you're ready  install python 3.7:
```
Comment (josephaxisa) [inline python/README.rst]: add 'to' before 'install'

File: python/README.rst [hunk 3]
```diff
@@ -16,14 +16,31 @@ and please indicate which language SDK you're using in the report.
...
+`pyenv <https://github.com/pyenv/pyenv#installation>`_ to install
+different versions of python. Mac users should use
+`homebrew <https://brew.sh/>`_ to install pyenv:
 
 .. code-block:: bash
 
-    brew update && brew install pyenv && brew install pipenv
+    brew install pyenv
+
+Follow the **remaining steps 3 - 5** of
```
Comment (josephaxisa) [inline python/README.rst]: Might be worth numbering the steps if we're referring to them like this.

File: python/README.rst [hunk 4]
```diff
@@ -36,14 +53,13 @@ Install python3.7 and use it for this directory
 .. code-block:: bash
```
Comment (josephaxisa) [inline python/README.rst]: I think L34 should read something along the lines of, "Set the local Python version for our project to 3.7.4"

File: python/README.rst (hunk 3 above)
Comment (joeldodge79) [reply python/README.rst]: this is in reference to the link (https://github.com/pyenv/pyenv#basic-github-checkout) which does have the steps numbered :-)

File: python/README.rst (hunk 2 above)
Comment (joeldodge79) [inline python/README.rst]:
```suggestion
Now you're ready to install python 3.7:
```

File: python/README.rst (hunk 1 above)
Comment (joeldodge79) [reply python/README.rst]: I don't think there's a reason to launch them into `pipenv shell` I'd rather stick with a one-liner

**Pull request merged**